doc/source/user_guide/ug_implementation.rst

```diff
@@ -197,7 +199,8 @@ written, the USE_NETCDF C preprocessor directive must be set during compilation.
 is done by setting ``ICE_IOTYPE`` to ``netcdf`` in **icepack.settings**.  In addition,
 the machine env and Macros files must include support for compilation with NetCDF.  The
 ``icepack.setup -s`` option ``ionetcdf`` will set the ICE_IOTYPE to netcdf, which turns on 
-the USE_NETCDF C preprocessor.  ``ionetcdf`` also sets the ``history_cdf`` flag to true.
+the USE_NETCDF C preprocessor.  ``ionetcdf`` also sets the ``history_format`` and 
+``restart_format`` flags to 'nc'.
```

Comment:
It looks like set_nml.ionetcdf is empty now.   Is that true?

Reply:
Correct, will fix that on Consortium version as well.